common/Cargo.toml

```diff
@@ -58,7 +57,7 @@ siphasher = "0.3"
 backtrace = { version = "0.3", optional = true }
 chrono = { version = "0.4", optional = true }
 mc-util-logger-macros = { path = "../util/logger-macros", optional = true }
-slog = { version = "2.5", features = ["dynamic-keys", "max_level_trace", "release_max_level_trace"], optional = true }
+slog = { version = "2.6", default-features = false, features = ["dynamic-keys", "max_level_trace", "release_max_level_trace"] }
```

Comment:
Do all of these features (`dynamic-keys`, `max_level_trace`, `release_max_level_trace`) need to be enabled by default or should they be enabled in the `log` feature?

Reply:
i think it may be confusing if `slog` is configured one way in the enclave and a different way in the logging sync that handles the serialized slog logs outside the enclave.

i suspect that the build will break if `dynamic-keys` is set differently, and i can't easily predict what will happen if `max_level_trace` or `max_release_level_trace` is different in different places